utils/scripts/install-el8.sh

```diff
@@ -61,6 +61,13 @@ dnf --nodocs install \
     which \
     yasm
 
+# ipmctl is only available on x86_64
+if [[ $arch == x86_64* ]]; then
```

Comment:
(lint) In POSIX sh, [[ ]] is undefined. [SC3010]

Reply:
can probably just make this #!/bin/bash